Kommunicate-io/Kommunicate-Web-SDK
# FW-1960 Iframe support in chat widget

### What do you want to achieve?
- To create a route which will be provided as a source for iframe.
- The iframe with new source will open the chat widget in the website of the customer.

### PR Checklist


### How was the code tested?

- Create a test HTML page and append the following iframe in the page.
```
 <iframe 
            style="border: none;"
            height="600px"
            width="400px"
            src="HOST_URL/chat?appId=APP_ID"
            allow="microphone; geolocation;"
        >
    </iframe>
```
- Please change the HOST_URL and APP_ID with suitable values.
- Open the page in your web browser.
- The chat widget should load and work fine.

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
